cluster/clean.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+echo 'Cleaning up ...'
+
+./cluster/kubectl.sh delete --ignore-not-found -f deploy/
```

Comment:
no file specified

Reply:
This should deploy everything under the directory.